be/src/storage/push_handler.cpp

```diff
@@ -36,13 +36,14 @@ Status PushBrokerReader::init(const TBrokerScanRange& t_scan_range, const TDescr
     fragment_params.protocol_version = InternalServiceVersion::V1;
     TQueryOptions query_options;
     TQueryGlobals query_globals;
+    query_globals.__set_time_zone(request.time_zone);
```

Comment:
Should check timezone is set or not in request, because BE will upgrade first, then FE.